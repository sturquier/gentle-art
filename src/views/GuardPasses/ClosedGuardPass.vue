<template>
  <h1>Closed guard pass</h1>
</template>
