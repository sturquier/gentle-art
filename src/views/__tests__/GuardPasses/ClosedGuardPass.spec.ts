import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import ClosedGuardPass from '../../GuardPasses/ClosedGuardPass.vue'

describe('ClosedGuardPass', () => {
  it('renders properly', () => {
    const wrapper = mount(ClosedGuardPass)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
