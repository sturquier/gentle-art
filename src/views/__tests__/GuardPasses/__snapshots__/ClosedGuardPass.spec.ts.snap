// Vitest Snapshot v1

exports[`ClosedGuardPass > renders properly 1`] = `"<h1>Closed guard pass</h1>"`;
